app/src/main/java/com/wafflestudio/siksha2/compose/ui/community/CommentDetailDialog.kt

```diff
@@ -0,0 +1,117 @@
+package com.wafflestudio.siksha2.compose.ui.community
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Divider
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Dialog
+
+@Composable
+fun CommentDetailDialog(
+    onDismissRequest: () -> Unit,
+    onClickReply: () -> Unit,
+    onClickReport: () -> Unit,
+    onClickCancel: () -> Unit
+) {
+    Dialog(onDismissRequest = onDismissRequest) {
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = Modifier
+                .width(315.dp)
+                .height(165.dp)
+                .background(Color.White, shape = RoundedCornerShape(26.dp))
+                .border(
+                    width = 1.dp,
+                    color = Color.Gray,
+                    shape = RoundedCornerShape(26.dp)
+                )
+        ) {
+            Surface(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .weight(1f)
+                    .background(Color.White, RoundedCornerShape(26.dp)),
+                shape = RoundedCornerShape(26.dp),
+                color = Color.White
```

Comment:
```suggestion
                modifier = Modifier
                    .fillMaxWidth()
                    .weight(1f)
```

clip으로 하자고 한 것 관련해서 컴포넌트 Surface의 modifier는 이렇게 수정하면 될 듯